### DISCUSSION TITLE
How to create a SuperStream with partitions?

### DISCUSSION THREAD

**Discussion by anschauanderson** [rabbitmq/rabbitmq-stream-dotnet-client#278] [General]
I am able to write and consume superstream messages and it works for me.

But when i started de RabbitMQ i need execute a command 

rabbitmq-streams add_super_stream MyStreams --partitions 10

How can i create this programmatically when my pod starts?

**Comment by Gsantomaggio** [comment]
There is no way to create a super-stream programmatically. However, you can use the HTTP API to create it or import the "load definition" feature. 

Do you use the https://github.com/rabbitmq/cluster-operator ?

**Comment by acogoluegnes** [comment]
It is also possible to use plain AMQP to declare the exchange, streams, and bind them together. You have to include and use an AMQP client library though.

**Comment by anschauanderson** [comment]
Thank you very much.
